Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Tabard of the Hand of Argus

## Changelog:
- Added a tabard for the Hand of Argus. Giving it to Kuros, Boros, and Aesom.




# How to test:
- Start game at any date
- Wait for Draenei to appear (or use console: `event WCDRN.99`)
